web/server/README.md

```diff
@@ -91,6 +91,28 @@ Netdata supports access lists in `netdata.conf`:
 
 - `allow management from` checks the IPs to allow API management calls. Management via the API is currently supported for [health](../api/health/#health-management-api)
 
+### Enabling TLS/SSL support
+
+Netdata since version 1.14 supports encrypted connections. To enable this option it is necessary to genereate a certificate and a key, case you do not have one it is possible to generate them  with the following command
```

Comment:
since 1.15 i guess

Reply:
You are correct, I already fixed here and I will update in the next merge.